projects/aca-content/src/lib/components/sidenav/user-menu/user-menu.component.ts

```diff
@@ -50,9 +50,26 @@ export class UserMenuComponent implements OnInit {
   @Input()
   data: { items: any[] };
 
+  @ViewChild(MatMenu)
+  menu: MatMenu;
+
+  @ViewChildren(ToolbarMenuItemComponent)
+  toolbarMenuItems: QueryList<ToolbarMenuItemComponent>;
+
   ngOnInit() {
     if (this.data?.items) {
       this.data.items.sort((a, b) => a.order - b.order);
     }
   }
+
+  ngAfterViewInit() {
+    const menuItems: MatMenuItem[] = this.toolbarMenuItems
+      .map((toolbarMenuItem: ToolbarMenuItemComponent) => toolbarMenuItem.menuItem)
```

Comment:
Do we need to use map? I guess we can use only filter like: 
.filter((toolbarMenuItem) => toolbarMenuItem.menuItem !== undefined);

When using both we need to iterate toolbarMenuItems twice, but when we use only filter then we will iterate once